libmscore/ottava.cpp

```diff
@@ -173,25 +173,21 @@ Sid Ottava::getPropertyStyle(Pid pid) const
             Sid::ottava22MBText,
             };
 
+      int idx = int(_ottavaType) * 2 + (_numbersOnly ? 0 : 12);
       switch (pid) {
             case Pid::OFFSET:
                   return placeAbove() ? Sid::ottavaPosAbove : Sid::ottavaPosBelow;
-            case Pid::PLACEMENT: {
-                  int idx = int(_ottavaType) * 2 + (_numbersOnly ? 0 : 12);
+            case Pid::PLACEMENT:
                   return ss[idx];
-                  }
             case Pid::BEGIN_TEXT:
-            case Pid::CONTINUE_TEXT: {
-                  int idx = int(_ottavaType) * 2 + (_numbersOnly ? 0 : 12);
                   return ss[idx+1];       // BEGIN_TEXT
-                  }
-            case Pid::END_HOOK_HEIGHT: {
-                  int idx = int(_ottavaType) * 2 + (_numbersOnly ? 0 : 12);
+            case Pid::CONTINUE_TEXT:
+                  return ss[idx+3];       // CONTINUE_TEXT
```

Comment:
Maybe I'm missing something, but I don't see how this would work.  idx+3 seems like it would work out the "octave below" style setting.  The whole problem is there *is* no continue text style setting

Reply:
Right, found that out myself meanwhile